doc/reference/storage_ceph.md

```diff
@@ -1,56 +1,110 @@
-  It is recommended to not maintain any non-LXD owned filesystem entities in
-  a LXD OSD storage pool since LXD might delete them.
-- Note that sharing the same osd storage pool between multiple LXD instances is
-  not supported. LXD only allows sharing of an OSD storage pool between
-  multiple LXD instances only for backup purposes of existing instances via
-  `lxd import`. In line with this, LXD requires the "ceph.osd.force_reuse"
-  property to be set to true. If not set, LXD will refuse to reuse an osd
-  storage pool it detected as being in use by another LXD instance.
-- When setting up a Ceph cluster that LXD is going to use we recommend using
-  `xfs` as the underlying filesystem for the storage entities that are used to
-  hold OSD storage pools. Using `ext4` as the underlying filesystem for the
-  storage entities is not recommended by Ceph upstream. You may see unexpected
-  and erratic failures which are unrelated to LXD itself.
-- To use Ceph osd pool of type "erasure" you __must__ have the osd pool created
-  beforehand, as well as a separate osd pool of type "replicated" that will be used for
-  storing metadata. This is required as RBD & CephFS do not support omap.
-  To specify which pool is "earasure coded" you need to use the
-  `ceph.osd.data_pool_name=<erasure-coded-pool-name>` and
-  `source=<replicated-pool-name>` for the replicated pool.
-
-## Storage pool configuration
+# Ceph RBD - `ceph`
+
+<!-- Include start Ceph intro -->
+
+```{youtube} https://youtube.com/watch?v=kVLGbvRU98A
+```
+
+[Ceph](https://ceph.io/en/) is an open-source storage platform that stores its data in a storage cluster based on {abbr}`RADOS (Reliable Autonomic Distributed Object Store)`.
+It is highly scalable and, as a distributed system without a single point of failure, very reliable.
+
+Ceph provides different components for block storage and for file systems.
+<!-- Include end Ceph intro -->
+
+Ceph {abbr}`RBD (RADOS Block Device)` is Ceph's block storage component that distributes data and workload across the Ceph cluster.
+It uses thin provisioning, which means that it is possible to over-commit resources.
+
+## Terminology
+
+<!-- Include start Ceph terminology -->
+Ceph uses the term *object* for the data that it stores.
+The daemon that is responsible for storing and managing data is the *Ceph {abbr}`OSD (Object Storage Daemon)`*.
+Ceph's storage is divided into *pools*, which are logical partitions for storing objects.
+They are also referred to as *data pools*, *storage pools* or *OSD pools*.
+<!-- Include end Ceph terminology -->
+
+Ceph block devices are also called *RBD images*, and you can create *snapshots* and *clones* of these RBD images.
+
+## `ceph` driver in LXD
+
+```{note}
+To use the Ceph RBD driver, you must specify it as `ceph`.
+This is slightly misleading, because it provides only Ceph RBD (block storage) functionality, not full Ceph functionality.
```

Comment:
"Therefore, the `ceph` driver can only be used for storage volumes with content type `block`." - This isn't correct.

While true that the `ceph` storage pool type only uses ceph's underlying block storage functionality, LXD will use ceph RBD block volumes to provide LXD filesystem volumes (for images, containers and custom filesystem volumes), as well as LXD block volumes (for VM images, VMs and custom block volumes).

Instead I propose we change it to something like:

> The `ceph` pool type only uses Ceph RBD (block storage) functionality, not full Ceph functionality.
LXD will create RBD volumes for both filesystem and block LXD volumes.
> It is also possible to use CephFS functionality for custom filesystem volumes by creating a {ref}`storage-cephfs` type pool.

Reply:
So `ceph` pool type actually supports all LXD functionality, whereas `cephfs` pool type only supports LXD custom filesystem volumes.